### ISSUE TITLE
answer<3

### ISSUE THREAD

**Issue by Its-Vichy** [dropout1337/question#3]
Hello my friend,
Discord is a social network that is killing our beautiful community by banning people for no reason and repeating.
Create a discord server with the big brains of this platform could be a good idea. Otherwise the server will be banned directly.

A big part of the community is on guilded, or telegram but unfortunately not everyone is on guilded :(
I also find that in general the community that is exclusively on discord are mentally limited and don't know how to search by themselves. 

Basically, either a server more or less closed or else they will get banned or they will have to go to another platform, or even host their own server revolt or fosscord.

<3

**Comment by dropout1337** [comment]
thanks for striking an idea in my head :)
